spanner/google/cloud/spanner/database.py

```diff
@@ -389,8 +389,7 @@ def batch(self):
         """
         return BatchCheckout(self)
 
-    def snapshot(self, read_timestamp=None, min_read_timestamp=None,
-                 max_staleness=None, exact_staleness=None):
+    def snapshot(self, **kw):
```

Comment:
@jonparrott I don't really care if pylint doesn't like it:  it find that tool over-opinionated.  `**kw` is *still* idiomatic Python:

- The 3.6.0 library still uses it in 694 cases,
- Our own code uses it 430 times.  Asking for changes here (rather than making a sweeping change through the codebase) isn't the right way to make such changes.